package.json

```diff
@@ -257,6 +258,8 @@
     "eslint-plugin-wpcalypso": "3.4.1",
     "glob": "7.0.3",
     "husky": "0.13.3",
+    "ignore": "3.3.5",
```

Comment:
why are these two here?

Reply:
The `bin/reformat-files.js` script uses them. `glob` and `ignore` are used to list the JS and JSX files in the root directory and apply `.eslintignore` to the list. `jest-docblock` is used to check if the file has the `@format` tag in the initial docblock comment.